packages/graphiql-react/src/editor/query-editor.tsx

```diff
@@ -148,6 +148,8 @@ export function useQueryEditor(
         return;
       }
 
+      let showingHints = false;
```

Comment:
Nit: Can we move that variable down where we define the handlers to have better coloaction?